src/Sql/ValueFormatter.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\ClickHouseClient\Sql;
+
+use DateTime;
+use DateTimeImmutable;
+use DateTimeZone;
+use SimPod\ClickHouseClient\Exception\UnsupportedValueType;
+use function array_map;
+use function date_default_timezone_get;
+use function implode;
+use function is_array;
+use function is_float;
+use function is_int;
+use function is_object;
+use function is_string;
+use function method_exists;
+use function Safe\preg_match;
+use function Safe\sprintf;
+
+final class ValueFormatter
+{
+    /**
+     * @param mixed $value
+     */
+    public static function format($value, ?string $paramName = null, ?string $sql = null) : string
+    {
+        if (is_string($value)) {
+            return "'" . $value . "'";
+        }
+
+        if (is_int($value)) {
+            return (string) $value;
+        }
+
+        if (is_float($value)) {
+            return (string) $value;
+        }
+
+        if ($value === null) {
+            return 'IS NULL';
+        }
+
+        if ($value instanceof DateTimeImmutable || $value instanceof DateTime) {
+            return "'" . $value
+                    ->setTimezone(new DateTimeZone(date_default_timezone_get()))
```

Comment:
proč ta timezone?

Reply:
clickhouse nema datetime s timezone. takze to musis znormalizovat, jinak ji urizne

ale mozna bych mel nejak vyresit injectovani timezone z venku

Reply:
imo by jsi měl mít v options client timezone ... a když se nepředá, tak použít tuhle no

Reply:
tohle je neprijemny, chtel jsem zachovat tu statiku, aby se to vsude nemuselo cpat skrz DI 🤔 musim premyslet